linera-core/src/worker.rs

```diff
@@ -811,7 +815,7 @@ where
     }
 
     /// Processes a confirmed block certificate.
-    #[instrument(skip_all, fields(
+    #[instrument(skip_all,  fields(
```

Comment:
typo

Reply:
Oh I was hoping `cargo fmt` would fix this one for me :sweat_smile: